### ISSUE TITLE
sections in a room

### ISSUE THREAD

**Issue by joshxfi** [joshxfi/trackAsOne#80]
a feature where u can create different sections in a room.

eg.
a section for activities & another section for announcements

**Comment by CheeTheStudent** [comment]
Hi, I'm kinda a newbie but I'm interested in your project and wanna contribute! How do you envision the sections of the room to be? Will users just be able to make as many sections as they wish and label them? I would like to try this out!

**Comment by joshxfi** [comment]
Thanks for your interest @CheeTheStudent ! Basically, I want to add another button here (encircled in the photo below). When you click that button, a modal will popup with the list of sections and an `Add Section` button.

We can check what section the room is on based on the URL. eg: `&section=announcements`

When adding a new section, it will be stored on the `sections` field (which is a string array) on the room's document.

When adding a task, u can add a `section` field on it to determine which section to show it.

As for the limit, I plan to have 4 extra sections at max.

![Screenshot_20220619-213511_Chrome.jpg](https://user-images.githubusercontent.com/69457996/174483819-cd28700b-6cc2-47b0-aaf0-3d9a8c38b0ce.jpg)

**Comment by CheeTheStudent** [comment]
Thanks for clarifying! I'm assuming the user can view one section at a time only, and select the section by clicking on the circled button you drew? 

I would love to try doing this feature. Should I contact you here if I have any questions?

**Comment by joshxfi** [comment]
Yes, that's correct. If you have questions, you can contact me on discord `!xfi#1387` or by mail `joshxfi.dev@gmail.com`
